### DISCUSSION TITLE
Discussions on specific implementations and WIP

### DISCUSSION THREAD

**Discussion by friedPotat0** [friedPotat0/Spam-Scores#41] [Contributors]
Please share ideas and problems about specific implementations and what you are currently working on here.

**Comment by dlh2** [comment]
If I think properly about the code itself, I don't quite understand the reason behind we need to restart the application to make it work.

At least by the tests we made so far, it shouldn' t be needed, therefore there's something wrong with the application of the legacy code (which we can't really get good docs about it).

Also because the addon was made for spamd & Spam Assassin, I can't really appreciate if it works or not, as when we load it in one folder it goes well but if I have another folder and move to that folder it won't show anymore, and I need to reload the module (because in debug mode you can't restart thunderbird)

I'll give it a go to change it in a way it even loads when moving from folders, then I don't think it's necessary neither to repair the folder not even restart.
